### PR TITLE
feat(auto): i forget what else i put in these commits honestly

### PR DESCRIPTION
**Pull request by TheGamer1002**

### CODE REVIEW

File: src/main/deploy/pathplanner/autos/Copy of bot-2.auto
Comment (NyxAlexandra) [inline src/main/deploy/pathplanner/autos/Copy of bot-2.auto:1]: is this used? if not, delete

File: src/main/java/frc/robot/commands/AutoShootCommand.kt [hunk 1]
```diff
@@ -0,0 +1,56 @@
+package frc.robot.commands
+
+import edu.wpi.first.wpilibj2.command.Command
+import frc.robot.subsystems.shooter.Shooter
+import java.util.function.DoubleSupplier
+
+/**
+ *
+ */
```
Comment (NyxAlexandra) [inline src/main/java/frc/robot/commands/AutoShootCommand.kt:9]: remove :P

File: src/main/java/frc/robot/commands/AutoShootCommand.kt [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    val maxRpm = 3500.0
+    init {
+        // each subsystem used by the command must be passed into the
+        // addRequirements() method (which takes a vararg of Subsystem)
+        addRequirements(this.shooter)
+    }
+
+    /**
+     *
+     */
```
Comment (NyxAlexandra) [inline src/main/java/frc/robot/commands/AutoShootCommand.kt:26]: remove :3

File: src/main/java/frc/robot/commands/AutoShootCommand.kt [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    override fun initialize() {
+    }
+
+    /**
+     *
+     */
```
Comment (NyxAlexandra) [inline src/main/java/frc/robot/commands/AutoShootCommand.kt:32]: remove :)

File: src/main/java/frc/robot/commands/AutoShootCommand.kt [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+        if(speed > 100) {
+            shooterIsRunning()
+        } else {
+            shooterIsNotRunning()
+        }
+    }
+
+    /**
+     *
+     */
```
Comment (NyxAlexandra) [inline src/main/java/frc/robot/commands/AutoShootCommand.kt:44]: remove :O

File: src/main/java/frc/robot/commands/AutoShootCommand.kt [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+    override fun isFinished(): Boolean {
+        // TODO: Make this return true when this Command no longer needs to run execute()
+        return false
+    }
+
+    /**
+     *
+     */
```
Comment (NyxAlexandra) [inline src/main/java/frc/robot/commands/AutoShootCommand.kt:52]: remove :>

File: src/main/java/frc/robot/util/LimelightHelpers.kt [hunk 6]
```diff
@@ -100,7 +178,6 @@ object LimelightHelpers {
 
     /////
     /////
```
Comment (NyxAlexandra) [inline src/main/java/frc/robot/util/LimelightHelpers.kt:180]: yk what I'm gonna put :P

File: src/main/java/frc/robot/util/LimelightHelpers.kt [hunk 7]
```diff
@@ -280,18 +378,21 @@ object LimelightHelpers {
         return toPose2D(result)
     }
 
-    @JvmStatic
     fun getTV(limelightName: String?): Boolean {
         return 1.0 == getLimelightNTDouble(limelightName, "tv")
     }
 
     /////
     /////
```
Comment (NyxAlexandra) [inline src/main/java/frc/robot/util/LimelightHelpers.kt:386]: why are there so many of these

**Review by NyxAlexandra — CHANGES_REQUESTED** [review]
a few changes please :)
pweaaaaassssseeeeee

**Comment by TheGamer1002** [comment]
it auto-added those im not gonna do anything about it part because im lazy and part because the ide yells at me otherwise

**Comment by NyxAlexandra** [comment]
> it auto-added those im not gonna do anything about it part because im lazy and part because the ide yells at me otherwise

>:(

**Comment by TheGamer1002** [comment]
okay im just gonna force merge it because i dont have time to make changes

**Review by NyxAlexandra — APPROVED** [review]
IIWIW ig, begrudgingly approved

**Pull request merged**